src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -37,7 +37,8 @@ final case class ClusterRequest(labels: LabelMap,
                           defaultClientId: Option[String] = None,
                           jupyterDockerImage: Option[String] = None,
                           rstudioDockerImage: Option[String] = None,
-                          scopes: Set[String] = Set.empty)
+                          scopes: Set[String] = Set.empty,
+                          enableWelder: Option[Boolean] = None)
```

Comment:
I'll make this default to `false`. At call site, you'll branching out based on only `true`, `false` anyway.

Reply:
It needs to be an `Option[Boolean]` to denote an optional json field. I think defaulting to None is appropriate for Option.